assets/stylesheets/components/_team.scss

```diff
@@ -1,42 +1,44 @@
-.team {
-
-	.member {
-		@include box;
-		float: left;
-		width: (100% / 3);
-		padding: 10px;
-		text-align: center;
-		border-left: solid 1px #ccc;
-		border-top: solid 1px #ccc;
```

Comment:
yes and we need to use vars for colors...but his can be merge this and worry about that late